<html>
    <script>
        try {
            throw new Error('thirdErrorExcluded');
        } catch (error) {
            console.log(error);
        }

        try {
            const _undefined = undefined;
            _undefined.map(() => true);
        } catch (error) {
            console.error(error);
        }
    </script>
</html>
